syntax = "proto3"; 	//指明使用proto3语法

package login_proto;

message user_login_request
{
    string account=1;
    string password=2;
}

message user_login_reply
{
    bool issuccess=1;
    repeated string msg=2;
}